### ISSUE TITLE
Top layers effects bottom layers

### ISSUE THREAD

**Issue by Thisismydigitalself** [Ultimaker/Cura#17432]
### Cura Version

5.5

### Operating System

Win10

### Printer

TLM

### Reproduction steps

These holes starts 1mm from bottom.
![3](https://github.com/Ultimaker/Cura/assets/30949968/83cc35cd-e27e-44e0-ab61-6fb7ed952c53)
This is how first layers looks like when top layers are less than 3.
![32](https://github.com/Ultimaker/Cura/assets/30949968/c178411f-057a-442d-808d-01e239efd683)
This is how first layers looks like when top layers are set to 5
![5](https://github.com/Ultimaker/Cura/assets/30949968/bdbafb52-3145-41fb-b3da-c41df0afc1e3)

Why Top surface skin layers effects the bottom layers?

### Actual results

Why Top surface skin layers effects the bottom layers?

### Expected results

Top layers should not effect bottom layers.

### Add your .zip and screenshots here ⬇️

These holes starts 1mm from bottom.
![3](https://github.com/Ultimaker/Cura/assets/30949968/83cc35cd-e27e-44e0-ab61-6fb7ed952c53)
This is how first layers looks like when top layers are less than 3.
![32](https://github.com/Ultimaker/Cura/assets/30949968/c178411f-057a-442d-808d-01e239efd683)
This is how first layers looks like when top layers are set to 5
![5](https://github.com/Ultimaker/Cura/assets/30949968/bdbafb52-3145-41fb-b3da-c41df0afc1e3)

Why Top surface skin layers effects the bottom layers?

**Comment by GregValiant** [comment]
Use the "File | Save Project" command, zip the 3mf file, and post the zip folder here.
It isn't possible to troubleshoot this without a project file.
It would appear that @5axes has seen this problem as well.  I haven't noticed it.

**Comment by 5axes** [comment]
Yes Top Skin Surface Layers takes precedence over Bottom Layer . I think we should have at least one Bottom Layer.
![image](https://github.com/Ultimaker/Cura/assets/11015345/86618320-2820-479c-b42b-b7344780e126)
[Cube_Hole.zip](https://github.com/Ultimaker/Cura/files/13468184/Cube_Hole.zip)

**Comment by tkocou** [comment]
I can confirm this problem. I changed the bottom surface layer from lines to concentric, but left the top surface as lines. After slicing the model and started the print, the bottom surface still printed with lines. I changed the top surface to be concentric as well. Sliced the model and the first layer printed concentric as intended.
